crates/database/src/ethflow_orders.rs

```diff
@@ -45,6 +45,30 @@ pub async fn read_order(
     sqlx::query_as(QUERY).bind(id).fetch_optional(ex).await
 }
 
+pub async fn mark_eth_orders_as_refunded(
+    ex: &mut PgTransaction<'_>,
+    uids: &[OrderUid],
+) -> Result<(), sqlx::Error> {
+    for uid in uids.iter() {
+        mark_eth_order_as_refunded(ex, uid).await?;
```

Comment:
Is there a specific reasoning why we don't update all rows in a single query?

Reply:
Because there is no simple way to pass an unknown number of parameters into a query.

Reply:
Ok. I was just wondering will the caller of `mark_eth_orders_as_refunded` be aware that partial update is possible. Maybe comment would suffice.

Reply:
In this case there won't be a partial update because the parameter is PgTransaction which means either all the calls succeed or the all fial.